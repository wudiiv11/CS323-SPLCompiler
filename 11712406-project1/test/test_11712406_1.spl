float c = 2;
int a

