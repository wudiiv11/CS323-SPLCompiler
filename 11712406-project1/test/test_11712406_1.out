Error type B at Line 1: assign value at global range!
Error type B at Line 1: Missing close brace
